.github/workflows/tests.yaml

```
@@ -31,15 +31,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kind-k8s-version: [1.28.15, 1.29.14, 1.30.10, 1.31.6, 1.32.3]
+        kind-k8s-version: [1.29.14, 1.30.10, 1.31.6, 1.32.3, 1.33.1]
```

We can update these to the latest [kindest node tags](https://hub.docker.com/r/kindest/node/tags) too:

```suggestion
        kind-k8s-version: [1.29.14, 1.30.13, 1.31.9, 1.32.5, 1.33.1]
```